connections/README.md

```diff
@@ -0,0 +1,9 @@
+# < Replace With The Plugin Name >
```

Comment:
We should fill out a basic readme for this.

Reply:
I've written up a README for the plugin. I am particularly interested in feedback from @swcurran on the wording and contents.